src/Http/WebUtilities/src/FormPipeReader.cs

```diff
@@ -31,6 +31,9 @@ public class FormPipeReader
         private PipeReader _pipeReader;
         private Encoding _encoding;
 
+        private static ReadOnlyMemory<byte> _otherEqualEncoding;
```

Comment:
Unfortunately this defeats the span optimizations. Let’s not do this. We can take the perf hit in the slow path

Reply:
This is only used for non utf8/ascii.

Reply:
Yes ignore my comment I didn't; see the other code path below.